p2p-tests/implicitly-tracked-peer-branches-test.ts

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env -S node --require ts-node/register/transpile-only --require tsconfig-paths/register
+
+// Copyright © 2021 The Radicle Upstream Contributors
+//
+// This file is part of radicle-upstream, distributed under the GPLv3
+// with Radicle Linking Exception. For full terms see the included
+// LICENSE file.
+
+import * as path from "path";
+import { strict as strictAssert } from "assert";
+
+import { sleep } from "ui/src/sleep";
+import {
+  RadicleProxy,
+  UpstreamSeed,
+  radCli,
+  runTestcase,
+  withRetry,
+} from "./lib/p2p";
+
+// Test that that the branches of contributors tracked by the maintainer are
+// replicated by any other peer.
```

Comment:
I think we can clarify this a bit and also mention that this currently fails.

```suggestion
// Test that when a peer clones a project it automatically adds all peers as remotes that the maintainer of the project has as remotes.
//
// This test case currently fails because the peer does not replicate branches for peers that the maintainer already tracks.
```

Reply:
My thinking was that we keep this PR open until we fix the issue and then commit the fix together with this test. I had forgotten to add it to `ci/p2p-network-tests.sh`, which I now did so it fails on CI. Does that make sense?

Reply:
Sounds good.